README.md

```diff
@@ -150,8 +150,33 @@ Please use GitHub’s issue tracker and pull request features.
 
 2. Install: `npm install`
 
-3. Use `npm run serve` to launch a local instance of the service. This will load
-   data from Cloud Storage; data can take up to a minute to be loaded and ready.
+3. Launch the local server:
+
+```zsh
+mkdir -p data/json
```

Comment:
This step is actually part of both paths one could take below, "Create /path/to/GoogleChrome/confluence/data/json" in the case of collecting data oneself.

Reply:
I'm not sure I quite follow...

Reply:
I mean that if this were code, there's the same thing (mkdir) both before and if-else, and in each branch.

Reply:
Ah. I see. Fixed.